site/homepage/content/release-notes.md

```diff
@@ -69,6 +69,10 @@ title = "Release Notes"
   because the Device Registration API does not define a corresponding operation.
   Consequently, the C&C functionality of the Kerlink Lora provider which relied on the *get*
   method has been removed.
+ 
+ ### Deprecations
+ 
+ * The configuration property `HONO_MQTT_COMMAND_ACK_TIMEOUT` of the MQTT adapter is now deprecated and planned to be removed in a future release. Use `HONO_MQTT_SEND_MESSAGE_TO_DEVICE_TIMEOUT` instead.
```

Comment:
Can you remove trailing whitespaces and add a linebreak so that it is aligned with the other lines?

Reply:
Done.